packages/command-palette/README.md

```diff
@@ -1,16 +1,16 @@
 # Command Palette
 
-The Command Palette is a UI element that displays a searchable list of commands. It can be accessed in Calypso using the shortcut `cmd+k`.
+The Command Palette is a UI element that displays a searchable list of commands. It can be accessed in WordPress.com using the shortcut `cmd+k`.
 
 This command palette is inspired by [Core's implementation](https://github.com/WordPress/gutenberg/blob/trunk/packages/commands/README.md) used in the Gutenberg editor. We use the same CSS and similar markup, the commands share the same basic props that we extended to support the following differences:
 
-- The Calypso version supports multisite commands with the sites as second step commands.
+- The WordPress.com version supports multisite commands with the sites as second step commands.
```

Comment:
Maybe we could explain this a bit better. Not all commands are multisite commands and in wp-admin context no commands will be multisite.

Reply:
I think that's unrelated to this PR. But anyway, I feel this is intended to explain a difference with the Core command palette, not how multisite commands are set up.